### ISSUE TITLE
No issue just a question.

### ISSUE THREAD

**Issue by PodeCaradox** [roy-t/AStar#37]
Hello,

first of all thanks for sharing your AStar implementation on github. Can you answer me why you also have a     
IList<IEdge> Incoming { get; }
in your INode interface, maybe i missing something in the Code?
From your implementation of the AStar i see you only use the Outgoing List for the algorithm, which is self explained.

**Comment by roy-t** [comment]
Hey,

Good observation. the incoming list is not used when calculating the path. But it makes it a lot easier to increase the traversal time to reach a specific node. Otherwise you would need another lookup structure to find all edges going to a specific node.

I guess this technically means that it doesn't have to be part of the INode interface! (I could leave it on the concrete `Node` class that implements INode. Which I didn't realize before.

**Comment by roy-t** [comment]
I'll keep this question open for now, to potentially address it in potential v4, as its a breaking change to remove these members.

**Comment by PodeCaradox** [comment]
Yeah that's fine, i will use your implementations in a different way for my Grid. The MinHeap is really nice :)
